site/app/templates/polls/ViewPoll.twig

```diff
@@ -64,7 +64,7 @@
                                 "style": "display: inline"
                             } only %}
                             {% if option.isCorrect() and poll.isReleaseAnswer()%}
-                                <span id='correct-tag'>&#10004;</span>
+                                <i class="fa-solid fa-square-check" id="correct-tag"></i>
```

Comment:
Oops, I didn't see this earlier.  In HTML, the `id` attribute should be unique.  You should use a `class` instead, since there's a possibility that there might be multiple correct answers.  Many browsers will render things properly if there are duplicate IDs, but that is technically undefined behavior as far as I am aware.

This might be why you weren't able to count them in Cypress.